---a/net/minecraft/block/BlockBreakable.java
+++b/net/minecraft/block/BlockBreakable.java
@@ -1,34 +1,19 @@
 package net.minecraft.block;
 
 import net.minecraft.block.material.Material;
-import net.minecraft.block.state.IBlockState;
-import net.minecraft.init.Blocks;
-import net.minecraft.util.BlockPos;
-import net.minecraft.util.EnumFacing;
-import net.minecraft.world.IBlockAccess;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class BlockBreakable extends Block
-{
+
+public class BlockBreakable extends Block {
+
     private boolean ignoreSimilarity;
     private static final String __OBFID = "CL_00000254";
 
-    protected BlockBreakable(Material materialIn, boolean ignoreSimilarityIn)
-    {
+    protected BlockBreakable(Material materialIn, boolean ignoreSimilarityIn) {
         super(materialIn);
         this.ignoreSimilarity = ignoreSimilarityIn;
     }
 
-    public boolean isOpaqueCube()
-    {
+    public boolean isOpaqueCube() {
         return false;
     }
 
-    @SideOnly(Side.CLIENT)
-    public boolean shouldSideBeRendered(IBlockAccess worldIn, BlockPos pos, EnumFacing side)
-    {
-        IBlockState iblockstate = worldIn.getBlockState(pos);
-        Block block = iblockstate.getBlock();
-
-        if (this == Blocks.glass || this == Blocks.stained_glass)
-        {
-            if (worldIn.getBlockState(pos.offset(side.getOpposite())) != iblockstate)
-            {
-                return true;
-            }
-
-            if (block == this)
-            {
-                return false;
-            }
-        }
-
-        return !this.ignoreSimilarity && block == this ? false : super.shouldSideBeRendered(worldIn, pos, side);
-    }
-}
+}
